pandas/core/computation/ops.py

```diff
@@ -189,9 +189,6 @@ def ndim(self) -> int:
 
 
 class Constant(Term):
-    def __init__(self, value, env, side=None, encoding=None) -> None:
-        super().__init__(value, env, side=side, encoding=encoding)
```

Comment:
Should not be needed, just calls the parent constructor (and uses `value` instead of `name`)